CONTRIBUTING.md

```diff
@@ -19,8 +19,7 @@ We use [Lerna](https://lernajs.io/) to manage the packages in this repository. Y
 
 1. Clone the repository
 1. Install dependencies using `npm install` in the root of the project
-1. Run `npm run build` in the root of the project once.
-1. Run `npm test` in the root of the project or in one of the package folders
+1. Run `npm test` in the root of the project or in one of the package folders (note: `before` running `npm run test:stryker` and `npm run test:stryker-api` you have to run `npm run build`)
```

Comment:
@kmdrGroch can we remove the `test:stryker` and `test:stryker-api` scripts? Those 2 are maintained in the root package.json, but those are the only once (added once for convenience). Suggestion:

> 1. Run `npm test` in the root of the project or in one of the package folders. Running `npm test` in the root will also build the packages. However, running `npm test` in a child package directory _will not build_ the package, so you probably need to run `npm run build` before doing that.

Reply:
well, sure, but since i have removed that branch i can't edit it :c

Reply:
It seems that you have also removed that repository: `unknown repository` 😅